pom.xml

```diff
@@ -98,6 +103,7 @@
                 <configuration>
                     <showDeprecation>true</showDeprecation>
                     <showWarnings>true</showWarnings>
+                    <release>${java.version}</release>
```

Comment:
As we need a JDK 11 to build we can use the javac release option : https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-release.html